modules/core/src/main/scala/org/scalasteward/core/application/Cli.scala

```diff
@@ -44,6 +46,21 @@ object Cli {
     val processTimeout = "process-timeout"
   }
 
+  implicit val mergeRequestApprovalsConfigArgument: Argument[MergeRequestApprovalRulesCfg] =
+    Argument.from("approvals_rule_name=required_approvals") { s =>
+      s.split(":").toList match {
```

Comment:
I split with `:` but I can switch to `=` like we do in env variables. I don't have any preferences.

Reply:
`=` makes more sense I guess so I changed it already.